scripts/components/help-modal/data.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+
+export const helperMessages = {
+    "columns": [
+        {
+            show: true,
+            title: "Gutter",
+            description: "Space between columns <br /> (also known as <i>container gutter</i>.",
+            icon: <svg width='54' height='54' xmlns='http://www.w3.org/2000/svg' xmlnsXlink='http://www.w3.org/1999/xlink'><defs><rect id='prefix__a' x='0' y='0' width='54' height='54' rx='3'/><mask id='prefix__b' maskContentUnits='userSpaceOnUse' maskUnits='objectBoundingBox' x='0' y='0' width='54' height='54' fill='#fff'><use xlinkHref='#prefix__a'/></mask></defs><g fill='none' fillRule='evenodd'><use stroke='#C8C8C8' mask='url(#prefix__b)' strokeWidth='2' strokeLinecap='round' strokeDasharray='3' xlinkHref='#prefix__a'/><rect fill='#007CBA' x='16' y='3' width='4' height='48' rx='1'/><rect fill='#007CBA' x='34' y='3' width='4' height='48' rx='1'/><rect stroke='#DCDCDC' x='3.5' y='3.5' width='11' height='47' rx='2'/><rect stroke='#DCDCDC' x='39.5' y='3.5' width='11' height='47' rx='2'/><rect stroke='#DCDCDC' x='21.5' y='3.5' width='11' height='47' rx='2'/></g></svg>,
```

Comment:
add this to our icons set so we can reuse this icons if necessary  https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/editor/icons.js

Reply:
Sure, but maybe they should be in a seperate object, something like `illustrations`, because they at the same time belong and don't belong with the other icons :D

Reply:
agreed you can put them in a same file but export a different object.